docs/sources/nodes.md

```diff
@@ -7,6 +7,9 @@ The node source adds an `A` record per each node `externalIP` (if not found, any
 It also adds an `AAAA` record per each node IPv6 `internalIP`.
 The TTL of the records can be set with the `external-dns.alpha.kubernetes.io/ttl` node annotation.
 
+Nodes marked as **Unschedulable** as per [core/v1/NodeSpec](https://pkg.go.dev/k8s.io/api@v0.31.1/core/v1#NodeSpec) are excluded.
+This avoid exposing Unhealthy, NotReady or SchedulingDisabled (cordon) nodes.
```

Comment:
the referenced doc is more clear, citing also the default seems to be important:

	// Unschedulable controls node schedulability of new pods. By default, node is schedulable.
	// More info: https://kubernetes.io/docs/concepts/nodes/node/#manual-node-administration
	// +optional
	Unschedulable [bool](https://pkg.go.dev/builtin#bool) `json:"unschedulable,omitempty" protobuf:"varint,4,opt,name=unschedulable"`

Reply:
Thanks for the info, i'll add this 😄